CHANGELOG.md

```diff
@@ -26,9 +26,9 @@ These changes are available in the [master branch](https://github.com/PrefectHQ/
 ### Fixes
 
 - Fix Docker storage not pulling correct flow path - [#968](https://github.com/PrefectHQ/prefect/pull/968)
-- Set Docker storage to pull master git branch until next version is released - [#968](https://github.com/PrefectHQ/prefect/pull/968)
```

Comment:
Is this delete intentional?

Reply:
Yeah

Reply:
@cicdw Had raised the point of not needing to include that in the changelog